crates/hyperswitch_connectors/Cargo.toml

```diff
@@ -18,6 +18,8 @@ dummy_connector = [
 ]
 
 [dependencies]
+chrono = "0.4"
```

Comment:
@bsayak03 @awasthi21 Prefer to use the `time` crate over `chrono` since we're using it everywhere else in our codebase...